### DISCUSSION TITLE
How to validate images outside of blocks.

### DISCUSSION THREAD

**Discussion by Tom-revere** [area17/twill#1947] [Q&A]
I have an image field I want to make required. As it is outside of a block, I am adding this to the Request.php file

```php
public function rulesForUpdate()
{
    return [
        'article_tile_image' => 'required'
    ];
}
```

However when saving, I get this error

`422 Unprocessable Content`

Is there any way to validate media fields outside of a block?

Thanks

**Comment by haringsrob** [comment]
Hey @Tom-revere,

You can use the following (replace cover with your field name).

```php
    public function rulesForUpdate()
    {
        return ['medias.cover' => 'required|array|min:1'];
    }
```
